src/latexify/ast_utils_test.py

```diff
@@ -114,25 +90,23 @@ def test_is_constant(value: ast.AST, expected: bool) -> None:
     assert ast_utils.is_constant(value) is expected
 
 
-@test_utils.require_at_most(7)
 @pytest.mark.parametrize(
     "value,expected",
     [
-        (ast.Bytes(s=b"foo"), False),
-        (ast.Constant("bar"), True),
-        (ast.Ellipsis(), False),
-        (ast.NameConstant(value=None), False),
-        (ast.Num(n=123), False),
-        (ast.Str(s="baz"), True),
-        (ast.Expr(value=ast.Num(456)), False),
+        (ast.Constant(value=b"foo"), False),
+        (ast.Constant(value="bar"), True),
+        (ast.Constant(value=...), False),
+        (ast.Constant(value=None), False),
+        (ast.Constant(value=123), False),
+        (ast.Constant(value="baz"), True),
+        (ast.Expr(value=ast.Constant(value=456)), False),
         (ast.Global(names=["qux"]), False),
     ],
 )
 def test_is_str_legacy(value: ast.AST, expected: bool) -> None:
```

Comment:
this test case as well

Reply:
Same with decorator @test_utils.require_at_most(7)

Reply:
Yes, removing every test function with decoration `@test_utils.require_at_most(7)` should be fine.

Reply:
@odashi  Yes, I removed all the test function decorated with @test_utils.require_at_most(7)

Reply:
@odashi  please check and approve the Checks :D